modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/DataValidation/ObjectValidationTabs.tsx

```diff
@@ -165,7 +185,6 @@ function TriggerEventContainer({disabled, eventTypes}: ITriggerEventProps) {
 				disabled={disabled}
 				label={Liferay.Language.get('event')}
 				options={eventTypes}
-				value={0}
```

Comment:
@cgmonte @MariMalta You will need to keep this to maintain the "On Submission" option always selected.